File: Web/Scripts/app/common/models/Circuit.ts
namespace LiveMonitor {
	export interface ICircuit {
		ani: string;
		siteId: string;
		scheduleId: string;
		circuitStatus: CircuitStatus;
		description: string;
		recordingLevel: string;
		deviceTypeId: number;
	}

	export class CircuitDto implements ICircuit {
		public ani: string;
		public siteId: string;
		public scheduleId: string;
		public circuitStatus: CircuitStatus;
		public description: string;
		public recordingLevel: string;
		public deviceTypeId: number;
	}

	export class LiveCircuitEntry implements ICircuit {
		public ani: string;
		public siteId: string;
		public scheduleId: string;
		public circuitStatus: CircuitStatus;
		public description: string;
		public recordingLevel: string;
		public deviceTypeId: number;
		public facilityName: string;
		public call: Call;
	}
}

File: Domain/DTOs/Circuit.cs
using System;

// ReSharper disable once CheckNamespace
namespace CpcLiveMonitor.Domain
{
	public class Circuit
	{
		public String Ani { get; set; }
		public String SiteId { get; set; }
		public String ScheduleId { get; set; }
		public String Description { get; set; }
		public String RecordingLevel { get; set; }
		public Call CurrentCall { get; set; }
	}
}